.github/workflows/ci.yml

```diff
@@ -21,10 +21,10 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.20
```

Comment:
Go versioning seems to be the issue here - found https://github.com/actions/setup-go/issues/326#issuecomment-1414855139 which ran into the same issue. The `1.20` resolves to 
``` 
Successfully set up Go version 1.2
go version go1.2.2 linux/amd64
 ```

This should be changed to `1.20.0` instead.  Unable to push directly to this branch but tested that it works by creating this dummy PR https://github.com/open-telemetry/otel-arrow/pull/64 and confirming all the collector component tests run now.

Reply:
AHA I hadn't seen this note and of course yes, that explained it, and the fix is already upstream.